applications/dashboard/models/class.invitationmodel.php

```diff
@@ -160,7 +160,7 @@ public function save($formPostValues, $settings = false) {
                 try {
                     $this->send($invitationID);
                 } catch (Exception $ex) {
-                    $this->Validation->addValidationResult('Email', sprintf(t('Although the invitation was created successfully, the email failed to send. The server reported the following error: %s'), strip_tags($ex->getMessage())));
+                    $this->Validation->addValidationResult('Email', sprintf(t('Invitation email error description'), strip_tags($ex->getMessage())));
```

Comment:
Same comment about your key. Also be careful about the double translation. Look at the docblock of `Gdn_Validation::addValidationResult()`. Please start looking into what methods take a translated string or do the translation themselves to familiarize yourself with our methods instead of depending learning this from a code review.